controls/SV-204415.rb

```diff
@@ -14,20 +14,23 @@
     If the "/etc/pam.d/system-auth" and "/etc/pam.d/password-auth" configuration files allow for password hashes other
     than SHA512 to be used, this is a finding.'
   desc 'fix', 'Configure the operating system to store only SHA512 encrypted representations of passwords.
-    Add the following line in "/etc/pam.d/system-auth":
-    pam_unix.so sha512 shadow try_first_pass use_authtok
-    Add the following line in "/etc/pam.d/password-auth":
-    pam_unix.so sha512 shadow try_first_pass use_authtok
-    Note: Manual changes to the listed files may be overwritten by the "authconfig" program. The "authconfig" program
-    should not be used to update the configurations listed in this requirement.'
+
+Add the following line in "/etc/pam.d/system-auth":
+     pam_unix.so sha512 shadow try_first_pass use_authtok
+
+Add the following line in "/etc/pam.d/password-auth":
+     pam_unix.so sha512 shadow try_first_pass use_authtok
+
+Note: Per requirement RHEL-07-010199, RHEL 7 must be configured to not overwrite custom authentication configuration settings while using the authconfig utility, otherwise manual changes to the listed files will be overwritten whenever the authconfig utility is used.'
```

Comment:
Only updating the fix text. The check text is the same and the describe logic can remain as is.

Reply:
Discussed on a group call with Eugene to determine that no changes are required for these controls as the check text is the same and the "authconfig" note was set as a separate requirement as [mentioned in this comment](https://github.com/mitre/redhat-enterprise-linux-7-stig-baseline/pull/189/files#r1170293635).